### ISSUE TITLE
Disable terminal dimensions overlay upon resize

### ISSUE THREAD

**Issue by maxgashkov** [blinksh/blink#1148]
It would be nice to have an option to disable terminal dimensions overlay when the screen is resized for some reason.

The overlay is quite distracting when blink is open with some other app side-by-side on iPadOS and the other app (browser for example) switches back and forth between text-input mode and regular one. In that case the helper bar pops under the screen which forces blink's window to resize too, which in turn shows overlay notification.

https://www.dropbox.com/s/z9lhtury3hvd3qb/Video%202020-11-13%2011%2043%2042.mov?dl=0

**Comment by yury** [comment]
@maxgashkov this is known iPadOS issue. Other apps behave the same (for instance, Telegram).

As a workaround please turn on shortcuts in Settings.app -> Keyboard -> Shortcuts toggle. Another thing you can do is to press `Cmd-O` to force focus out terminal.

**Comment by maxgashkov** [comment]
I'm aware that this behavior is caused by iPadOS itself. Me feature suggestion is about making the option to get rid of the prominent visual notification in blink. I'm OK with the fact that the terminal will redraw its window. But I'm sure that some of the users don't need an exact col x row dimensions popup every time this happens.

**Comment by yury** [comment]
Ok, got it now. Sorry. Thanks for suggestion.

One thing you can do right now is to lock terminal size. 3f tap and lock button. It will lock terminal size, so no resizes - no notifications.

**Comment by maxgashkov** [comment]
Wow, what sorcery is this? Thanks, and cheers for your work on this project!

**Comment by yury** [comment]
Yep, we are thinking how make functions more discoverable. Sorry

**Comment by aimerib** [comment]
I just bought Blink and have been playing around with themes and trying out connections to some of my hosts. I found this by accident, and I can’t tell you how glad I am that I did. A simple help page inside the app containing a list of all finger gestures, shortcuts, and commands would be helpful!

It took me 5 minutes to realize I had to type config to get to the settings. This is fine because I love the minimal interface, but having a greeting mentioning the help command would be also helpful.

+1 for making features more discoverable, because this app is amazing!

A suggestion of how this could be more user friendly:

Every new shell displays a greeting along the lines of: “Type help to get started”
In the help menu, a command for “gestures and commands” could bring up a list of all gestures and commands with descriptions.

Thank you for this app, and keep up the great work!

**Comment by javierdemartin** [comment]
@aimerib We'll think about how we can add some kind of guide/explanation in the settings or in the [docs](blink.sh/docks) so the experience is better. 

Just a heads up you can press `CMD + ,` to bring up the settings menu if you're using a keyboard. And also keep holding `CMD` anywhere in the app to see the available shortcuts.

**Comment by jmeosbn** [comment]
> One thing you can do right now is to lock terminal size.

Ah, I wish I knew this before (but glad I do now!!). I had previously tried the lock expecting it to fix the font size (disable zoom) and so I think I forgot about it when it did not.

One thing to add, is that on iPhone, the soft keyboard hides/reappears when switching away/to the app. This causes the screen to jump about for a second or so without this lock 🔒. With it, Blink feels a lot smoother. 😎
